sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!! @param [in] glat - geodetic latitude(deg)                                  
+!! @param [in] glong - geodetic longitude(deg)                                
+!! @param [in] stl - local apparent solar time(hrs)                           
+!! @param [in] f107a - 3 month average of f10.7 flux                          
+!! @param [in] f107 - daily f10.7 flux for previous day                       
+!! @param [in] ap - magnetic index(daily) or when sw(9)=-1. :                 
+!!                - array containing:                                         
+!!                  (1) daily ap                                              
+!!                  (2) 3 hr ap index for current time                        
+!!                  (3) 3 hr ap index for 3 hrs before current time           
+!!                  (4) 3 hr ap index for 6 hrs before current time           
+!!                  (5) 3 hr ap index for 9 hrs before current time           
+!!                  (6) average of eight 3 hr ap indicies from 12 to 33 hrs pr
+!!                         to current time                                    
+!!                  (7) average of eight 3 hr ap indicies from 36 to 59 hrs pr
+!!                         to current time                                    
+!! @param [in] ic initial point
+!!
+!! @author Hann-Ming Henry Juang
+      function vtst7(iyd,sec,glat,glong,stl,f107a,f107,ap,ic) 
+      use gettemp_mod, only: sw,swc,isw
+!x    common/csw/sw(25),swc(25),isw 
+      dimension ap(7),iydl(2),secl(2),glatl(2),gll(2),stll(2) 
+      dimension fal(2),fl(2),apl(7,2),swl(25,2),swcl(25,2) 
+      save 
+      data iydl/2*-999/,secl/2*-999./,glatl/2*-999./,gll/2*-999./ 
+      data stll/2*-999./,fal/2*-999./,fl/2*-999./,apl/14*-999./ 
+      data swl/50*-999./,swcl/50*-999./ 
+      vtst7=0 
+      if(iyd.ne.iydl(ic)) goto 10 
+      if(sec.ne.secl(ic)) goto 10 
+      if(glat.ne.glatl(ic)) goto 10 
+      if(glong.ne.gll(ic)) goto 10 
+      if(stl.ne.stll(ic)) goto 10 
+      if(f107a.ne.fal(ic)) goto 10 
+      if(f107.ne.fl(ic)) goto 10 
+      do 5 i=1,7 
+        if(ap(i).ne.apl(i,ic)) goto 10 
+    5 end do 
+      do 7 i=1,25 
+        if(sw(i).ne.swl(i,ic)) goto 10 
+        if(swc(i).ne.swcl(i,ic)) goto 10 
+    7 end do 
+      goto 20 
+   10 continue 
+      vtst7=1 
+      iydl(ic)=iyd 
+      secl(ic)=sec 
+      glatl(ic)=glat 
+      gll(ic)=glong 
+      stll(ic)=stl 
+      fal(ic)=f107a 
+      fl(ic)=f107 
+      do 15 i=1,7 
+        apl(i,ic)=ap(i) 
+   15 end do 
+      do 16 i=1,25 
+        swl(i,ic)=sw(i) 
+        swcl(i,ic)=swc(i) 
+   16 end do 
+   20 continue 
+      return 
+      end function vtst7                               
+
+!-----------------------------------------------------------------------
+!> Thermospheric portion of nrlmsise-00                              
+!!
+!!    See gtd7 for more extensive comments                              
```

Comment:
What is gtd7? Is that a subprogram? If so, use parens after the name, and it will automatically get a hyper-link. So:

`See gtd7() for more extensive comments.`